Sources/Extensions/Foundation/CalendarExtensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  CalendarExtensions.swift
+//  SwifterSwift
+//
+//  Created by Chaithanya Prathyush on 09/11/17.
+//  Copyright © 2017 SwifterSwift
+//
+
+import Foundation
+
+public extension Calendar {
+    /// SwifterSwift: Return the number of days in the month for a specified 'Date'.
```

Comment:
Please update doc to the following:
```swift
/// SwifterSwift: Number of days in the month for a specified Date.
///
///		let date = Date() // "Jan 12, 2017, 7:07 PM"
///		Calendar.current.numberOfDaysInMonth(for: date) -> 31
///
/// - Parameter date: Date from which the number of days in month is calculated.
/// - Returns: Number of days in the month for a specified Date object.
```

btw, you can use the shortcut `option` + `cmd` + `/` in Xcode to create the body for any given function :)